Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -120,6 +120,44 @@ def getDebugName(self, nameID):
 		else:
 			return None
 
+	def getFirstDebugName(self, nameIDs):
+		for nameID in nameIDs:
+			name = self.getDebugName(nameID)
+			if name is not None:
+				return name
+		return None
+
+	def getBestFamilyName(self):
+		# 21 = WWS Family Name
+		# 16 = Typographic Family Name
+		# 1 = Family Name
+		return self.getFirstDebugName((21, 16, 1))
+
+	def getBestSubFamilyName(self):
+		# 22 = WWS SubFamily Name
+		# 17 = Typographic SubFamily Name
+		# 2 = SubFamily Name
+		return self.getFirstDebugName((22, 17, 2))
```

Comment:
I don't think we should always priorise the record 22 and 17.

There should be a parameter. For example, in my case, I would like to get the "real" SubFamily Name.